### PR TITLE
lru-cache 7.5 typings

### PR DESCRIPTION
**Pull request by dhensby**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/isaacs/node-lru-cache/compare/v7.4.0...v7.5.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/lru-cache/index.d.ts
Comment (gabritto) [inline types/lru-cache/index.d.ts:9]: is `options` no longer optional starting at 7.5 or was it never optional and DT was wrong? I can't seem to find a change related to that on the original library (https://github.com/isaacs/node-lru-cache/compare/v7.4.0...v7.5.0)

**Review by gabritto — COMMENTED** [review]
this seems good, but let's wait for owner reviews since it's a popular package. thanks

File: types/lru-cache/index.d.ts
Comment (dhensby) [reply types/lru-cache/index.d.ts:9]: It was never optional and the typings are wrong :(

See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59162

**Comment by dhensby** [comment]
@gabritto unfortunately we aren't seeing much action from the designated code owners on this module (there have been several PRs now to these definitions without their feedback)

**Review by gabritto — APPROVED** [review]

**Comment by dhensby** [comment]
Ready to merge

**Pull request merged**
